http4s/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -153,6 +177,12 @@ class CollectorService[F[_]: Sync](
       }
     }
 
+  /** If the pixel is requested, this attaches cache control headers to the response to prevent any caching. */
+  def cacheControl(pixelExpected: Boolean): Option[`Cache-Control`] =
+    if (pixelExpected)
+      Some(`Cache-Control`(CacheDirective.`no-cache`(), CacheDirective.`no-store`, CacheDirective.`must-revalidate`))
```

Comment:
This line is very important!  I had forgotten the collector needs to do this, or else trackers will not send all requests to the collector.

Do we have a test somewhere (new unit test or old integration test) which checks these are set?

Reply:
I couldn't find any mention of that in the integration tests but there are some [checks](https://github.com/snowplow/stream-collector/blob/master/core/src/test/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorServiceSpec.scala#L217) for that in the unit tests. I've added specific unit test for that in the new application's test suite as well.